librosa/core/notation.py

```diff
@@ -402,15 +402,12 @@ def mela_to_svara(
         svara_map[10] = "Ni\u2082"
 
     if abbr:
-        svara_map = [
-            s.translate(str.maketrans({"a": "", "h": "", "i": ""})) for s in svara_map
-        ]
+        t_abbr = str.maketrans({"a": "", "h": "", "i": ""})
+        svara_map = [s.translate(t_abbr) for s in svara_map]
 
     if not unicode:
-        svara_map = [
-            s.translate(str.maketrans({"\u2081": "1", "\u2082": "2", "\u2083": "3"}))
-            for s in svara_map
-        ]
+        t_uni = str.maketrans({"\u2081": "1", "\u2082": "2", "\u2083": "3"})
+        svara_map = [s.translate(t_uni) for s in svara_map]
```

Comment:
these translator objects could probably be bumped up to the module level  / globals.